biography.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="eng">
+  <head>
+    <link rel="stylesheet" href="indexStyles.css" />
+    <link rel="stylesheet" href="biographyStyles.css" />
+  </head>
+  <body>
+    <!--Header Section-->
+    <section class="header">
+      <div class="link-container">
+        <a class="header-link" href="index.html">Home</a>
+      </div>
+      <div class="link-container">
+        <a class="header-link" href="biography.html" id="biography">Biography</a>
+      </div>
+      <div class="link-container">
+        <a class="header-link" href="experience.html">Experience</a>
+      </div>
+      <div class="link-container">
+        <a class="header-link" href="portfolio.html">Portfolio</a>
+      </div>
+      <div class="underliner">
+        <div class="logo">Portfol.io</div>
+      </div>
+    </section>
+    <!--Header Section-->
+
+    <!--Profile Section-->
+    <div class="main">
+      <section class="profile">
+        <img
+          src="https://avatars.githubusercontent.com/u/109642366?v=4"
+          alt="profile image"
+          class="profile-image"
+        />
+        <p class="about-me" id="signature">Eric Roberson</p>
+        <div class="social-links">
+          <p class="about-me">LinkedIn</p>
```

Comment:
nit: related to user experience, make sure that any links/buttons that look intuitively clickable (like links to socials) could be clicked.